scripts/metadata_comparison/extractor.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python3
+#
+# extractor.py
+#
+# Purpose: Read workflow metadata from Cromwell, and all metadata for its jobs,
+# and upload it to a GCS bucket
+#
+# Usage: python3 extractor.py <GCS path> <workflowId> [<workflowId2> [...]]
+#
+# Python Prereqs (at least, the ones which I needed to manually install... YMMV):
+#
+#   * pip3 install --upgrade requests
+#   * pip3 install --upgrade google-api-python-client
+#   * pip3 install --upgrade google-cloud
+#   * pip3 install --upgrade google-cloud-storage
+#   * pip3 install --upgrade google-api-python-client
```

Comment:
a dependency so nice you've installed it twice

Reply:
TOLish when I was using the python:3 image on Jenkins I wrote a script to install all dependencies like this and then run the main script.

Reply:
I'm I have a back-of-my-mind thought there's also some magic `__dependencies.py` or similar file you can put next to python scripts that auto-install dependencies when you're about to run the script